integration-libs/s4-service/order/components/s4-service-order-detail-actions/s4-service-order-detail-actions.component.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { ChangeDetectionStrategy, Component, inject } from '@angular/core';
+import { GlobalMessageService, GlobalMessageType } from '@spartacus/core';
+import { OrderDetailActionsComponent } from '@spartacus/order/components';
+import { Order } from '@spartacus/order/root';
+import { CheckoutServiceSchedulePickerService } from '@spartacus/s4-service/root';
+import { map, Observable } from 'rxjs';
+
+@Component({
+  selector: 'cx-s4-service-order-detail-actions',
+  templateUrl: './s4-service-order-detail-actions.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class S4ServiceOrderDetailActionsComponent extends OrderDetailActionsComponent {
+  private checkoutServiceSchedulePickerService = inject(
+    CheckoutServiceSchedulePickerService
+  );
+  private globalMessageService = inject(GlobalMessageService);
```

Comment:
```suggestion
  protected checkoutServiceSchedulePickerService = inject(
    CheckoutServiceSchedulePickerService
  );
  protected globalMessageService = inject(GlobalMessageService);
```
Apologies, only now I noticed that in the proposal I sent there were `private` access modifiers. Let's change it to `protected`.